src/zoo/owl_zoo_cmd.ml

```diff
@@ -92,11 +100,10 @@ let show_info gist =
 
 (* format "gist/file", e.g., d7bdd62b355f906ed059f00b1270b79c/readme.md *)
 let load_file f =
```

Comment:
With the new scheme, do you think this function still works properly? (because you are using subfolder, i cannot see it will work without using your extend_dir function.

Reply:
My bad. I'll fix this function.